### DISCUSSION TITLE
Typings problem with @types/select2 library

### DISCUSSION THREAD

**Discussion by xxxKOTxxx** [DefinitelyTyped/DefinitelyTyped#54004] [Issues with a @types package]
### Discussed in https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/53879

<div type='discussions-op-text'>

<sup>Originally posted by **xxxKOTxxx** June 17, 2021</sup>
Problem caused by next definition:

export interface DataFormat {
    id: number | string;
    text: string;
    selected?: boolean;
    disabled?: boolean;
}
As mentioned in this issue we can handle dropdown title by adding according property title. But currently it causes conflict with interface (same for html property). OptionData and OptGroupData are not suitable because has other semantic and contains required props like disabled, etc..

Please, update DataFormat interface by adding optional properties title?: string; and html?: string;. Possibly it has other undocumented properties?

If I need to use another interface, please mention it in comments.

Reproducing on "@types/select2": "^4.0.53"

Best regards, Borys</div>

**Comment by xxxKOTxxx** [comment]
Hi @orta, 
Why are You closed related issue?
I see that maintainers do not reply for discussions for a month... Maybe I doing it wrong?

**Reply by orta** [reply]
Discussions are for talking about the `@types` dependencies, not issues, I'd recommend pinging the authors or writing a PR to make the changes you're requesting

**Reply by xxxKOTxxx** [reply]
Thank You! 

I'll try to write a PR when found time for it!

Have a nice day!
